app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -80,32 +86,37 @@ fun HomeSectionHeader(
 @Composable
 private fun HomeSectionHeaderContent(
     headerText: String,
-    description: String,
+    textColor: Color = FirefoxTheme.colors.textPrimary,
+    description: String = "",
```

Comment:
We should avoid providing a default value for `description` and make this an explicit requirement.